CONTRIBUTING.rst

```diff
@@ -82,15 +89,15 @@ optimize performance.
    need to round, get the max or get the min of a series, there is no need to filter out nulls
    before doing that calculation.
 4. ``pd.to_numeric`` is preferred over ``as_type``.
-5. ``pd.to_numeric`` also replaces all None values with nans that can be operated on since ``np.nan``
-   is a float type.
+5. ``pd.to_numeric`` also replaces all None values with NaNs that can be operated on since
+   ``np.nan`` is a float type.
 6. If you are working with a series that has booleans and null values, there is a
    `nullable boolean type`_ that can be leveraged to avoid having to filter out null values.
 
 Create Performance Notebook
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once your transformer is complete, please follow the format in this sample `Collab Notebook`_
+Once your transformer is complete, please follow the format in this sample `Colab Notebook`_
```

Comment:
I think google actually only spells it with 1 'l', but since the actual word has two I'm fine with this